ext/zend_test/test.stub.php

```diff
@@ -109,6 +109,15 @@ function zend_test_compile_string(string $source_string, string $filename, int $
     /** @deprecated */
     function zend_test_deprecated(mixed $arg = null): void {}
 
+    /** @alias zend_test_alias */
```

Comment:
There was a problem with the aliases. If you run `./build/gen_stub.php --verify --force-regeneration` then gen_stub.php complains about the inexistent alias functions (`zend_test_alias`, `zend_test_deprecated_alias` etc.). So I fixed them in https://github.com/php/php-src/commit/ef21bbe66cdeb0cd829a9cea99c07c9c2c071b0a